forms-web-app/src/lib/appeals-api-wrapper.js

```diff
@@ -1,35 +1,75 @@
 const fetch = require('node-fetch');
+const uuid = require('uuid');
+const { utils } = require('@pins/common');
+
 const config = require('../config');
+const parentLogger = require('./logger');
+
+async function handler(url, method = 'GET', opts = {}, headers = {}) {
```

Comment:
Would renaming `url` to `path` be beneficial here? 

`url` implies I can / should pass in a full url, but that seems to break line 10

Reply:
Yeah, I'm easy either way with names. Will rename to `path` - the idea with this handle is that logging, authentication (if/when it comes) and routing are all handled automagically and the name of the variable makes no difference